### DISCUSSION TITLE
PYTHONPATH

### DISCUSSION THREAD

**Discussion by adrianportillojuan** [snl-dakota/dakota#61] [Q&A]
Hi everybody:

I am new in using Dakota. I have downloaded version 6.16.0 for windows (both UI and Command Line) and I am doing the cantilever initial tutorial for Sensitivity Analysis introduction and Sampling. When I run the input file as dakota it shows that it cannot find "Python".

I have checked the installation and apparently this can be due to the environment variables, however, I already defined PYTHONPATH variable and have Python downloaded in my computer. Can someone help me with this issue from their experience? That would be awesome.

Thank you very much in advance.

Adrian

**Comment by jadamstephens** [comment]
It's likely that Python is not on your Path.

There are a couple of ways to add it. Here's some instructions that I found just by googling: https://realpython.com/add-python-to-path/#how-to-add-python-to-path-on-windows
